dali/ft/filecopy.cpp

```diff
@@ -2950,9 +2954,12 @@ void FileSprayer::transferUsingAPI()
             ApiCopyStatus status = apiClient->getProgress(dateTime, outputLength);
             OutputProgress newProgress;
             newProgress.set(curProgress);
-            newProgress.resultTime = dateTime;
-            newProgress.outputLength = outputLength;
-            newProgress.inputLength = outputLength; // bytes read==bytes outputted
+            if (status != ApiCopyStatus::Failed && status!= ApiCopyStatus::Failed)
```

Comment:
so good you tested it twice?

Reply:
Had to make sure it was really sure! (I've changed one the test to check ::Aborted)